spatialpandas/geometry/point.py

```diff
@@ -1,9 +1,12 @@
 import numpy as np
 from pandas.core.dtypes.dtypes import register_extension_dtype
+from spatialpandas.geometry._algorithms.intersection import segment_intersects_point, \
+    point_intersects_polygon
 
 from spatialpandas.geometry.base import GeometryDtype
 from spatialpandas.geometry.basefixed import GeometryFixed, GeometryFixedArray
 from dask.dataframe.extensions import make_array_nonempty
+from spatialpandas.utils import ngpjit
```

Comment:
Should these be relative imports? Seems odd to see imports from "spatialpandas" inside spatialpandas.

Reply:
They could be, but this library is already Python 3 only (due to pandas dependency) so it works either way.